plugins/pasteembed/trumbowyg.pasteembed.js

```diff
@@ -0,0 +1,81 @@
+/* ===========================================================
+ * trumbowyg.pasteembed.js v1.0
+ * Url paste to iframe with noembed. Plugin for Trumbowyg
+ * http://alex-d.github.com/Trumbowyg
+ * ===========================================================
+ * Author : Max Seelig
+ *          Facebook : https://facebook.com/maxse
+ *          Website : https://www.maxmade.nl/
+ */
+
+(function($) {
+    "use strict";
+
+    $.extend(true, $.trumbowyg, {
+        plugins: {
+            pasteImage: {
+                init: function(trumbowyg) {
+                    trumbowyg.pasteHandlers.push(function(pasteEvent) {
+                        try {
+                            var clipboardData = (pasteEvent.originalEvent || pasteEvent).clipboardData;
+                            var pastedData = clipboardData.getData("Text");
+                            var request = null;
+
+                            if (pastedData.startsWith("http")) {
+
+                                pasteEvent.stopPropagation();
+                                pasteEvent.preventDefault();
+
+                                var query = {
+                                    url: pastedData.trim()
+                                };
+                                var content = "";
+                                var fails = 0;
+
+                                if (request && request.transport) request.transport.abort();
+
+                                request = $.ajax({
+                                    crossOrigin: true,
+                                    url: "https://noembed.com/embed?nowrap=on",
```

Comment:
How about having those urls configurable?
Would be great, so someone can use his own service to handle pasted urls.

Reply:
Let's do it :)